tgui/packages/tgui/interfaces/CommunicationsConsole/index.tsx

```diff
@@ -42,8 +42,9 @@ export function CommunicationsConsole(props) {
       break;
   }
 
+//NOVA CHANGE - ORIGINAL: width=400, height=650
   return (
-    <Window width={400} height={650} theme={emagged ? 'syndicate' : undefined}>
+    <Window width={450} height={750} theme={emagged ? 'syndicate' : undefined}>
```

Comment:
```suggestion
    <Window width={450} height={750} theme={emagged ? 'syndicate' : undefined}>
    //<Window width={400} height={650} theme={emagged ? 'syndicate' : undefined}> // NOVA EDIT REMOVAL
    <Window width={450} height={750} theme={emagged ? 'syndicate' : undefined}> { /* NOVA EDIT ADDITION */ }
```
Would prefer it to be tagged like this if you can't put it all on the same line, easier to keep track of that way.

Reply:
Sounds good